content/docs/04-ai-sdk-rsc/01-overview.mdx

```diff
@@ -175,17 +175,9 @@ const text = generateText({
       }),
       execute: async ({ city, unit }) => {
         const weather = getWeather({ city, unit })
-        const { temperature, unit, description, forecast } = weather
 
         uiStream.done(
-          <WeatherCard
-            weather={{
-              temperature: 47,
-              unit: 'F',
-              description: 'sunny'
-              forecast,
-            }}
-          />
+          <WeatherCard weather={weather} />
```

Comment:
this will affect the highlighting (line 158) which needs to be updated

Reply:
oh you are right, fixed